partitioned-table.md

```diff
@@ -356,24 +351,82 @@ After creating the partitions as above, you can easily add or delete records rel
 
 You can also execute `ALTER TABLE employees DROP PARTITION pEast` to delete all related rows, but this statement also deletes the `pEast` partition from the table definition. In this situation, you must execute the `ALTER TABLE ... ADD PARTITION` statement to recover the original partitioning scheme of the table.
 
-Unlike Range partitioning, List partitioning does not have a similar `MAXVALUE` partition to store all values that do not belong to other partitions. Instead, all expected values of the partition expression must be included in the `PARTITION ... VALUES IN (...)` clause. If the value to be inserted in an `INSERT` statement does not match the column value set of any partition, the statement fails to execute and an error is reported. See the following example:
+#### Default List partition
+
+Starting from v7.3.0, List partitioning supports the default partition feature, which lets you add a default List partition to a List partitioned table. The default List partition acts as a catch-all partition, where rows that do not match any other defined value sets can be placed.
+
+The default List partition feature is controlled by the [`tidb_enable_default_list_partition`](/system-variables.md#tidb_enable_default_list_partition-new-in-v730) variable, which is disabled by default because it is a TiDB extension and is not compatible with MySQL.
+
+When [`tidb_enable_default_list_partition`](/system-variables.md#tidb_enable_default_list_partition-new-in-v730) is set to `ON`, you can add a default partition to an existing List partition table.
```

Comment:
Global comment: please update all `List partition table` to `List partitioned table`.
```suggestion
When [`tidb_enable_default_list_partition`](/system-variables.md#tidb_enable_default_list_partition-new-in-v730) is set to `ON`, you can add a default partition to an existing List partitioned table.
```

Reply:
Done.